checks.d/fluentd.py

```diff
@@ -45,6 +46,9 @@ def check(self, instance):
                         continue
                     if p.get('plugin_id') in plugin_ids:
                         self.gauge('fluentd.%s' % (m), p.get(m), ["plugin_id:%s" % p.get('plugin_id')])
+
+                if p.get(self.RATE) is not None and p.get('plugin_id') in plugin_ids:
+                    self.rate('fluentd.%s' % (self.RATE), p.get(self.RATE), ["plugin_id:%s" % p.get('plugin_id')])
```

Comment:
Nitpick: You could rewrite this whole part with something like:
```python
for p in status['plugins']:
    for m_name, m_func in self.METRICS:
        if p.get(m_name) is None:
            continue
        m_func(self, 'fluentd.%s' % m_name, p.get(m_name), ["plugin_id:%s" % p.get('plugin_id')])
```

assuming that:
```python
METRICS = [
    ('retry_count', AgentCheck.gauge),
    ('buffer_queue_length', AgentCheck.gauge),
    ('buffer_total_queued_size', AgentCheck.rate),
]
```
it's slightly more concise and more readable.